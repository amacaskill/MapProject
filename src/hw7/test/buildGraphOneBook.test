# builds graph with different characters that are all only in one book

LoadGraph g buildGraphDifChars2.tsv

ListNodes g

ListChildren g char1
ListChildren g char2
ListChildren g char3
ListChildren g char4