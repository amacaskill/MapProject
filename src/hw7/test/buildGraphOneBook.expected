# builds graph with different characters that are all only in one book

loaded graph g

g contains: char1 char2 char3 char4

the children of char1 in g are: char2(1.000) char3(1.000) char4(1.000)
the children of char2 in g are: char1(1.000) char3(1.000) char4(1.000)
the children of char3 in g are: char1(1.000) char2(1.000) char4(1.000)
the children of char4 in g are: char1(1.000) char2(1.000) char3(1.000)